lib/ansiblelint/utils.py

```diff
@@ -241,7 +241,10 @@ def _taskshandlers_children(basedir, k, v, parent_type):
         if 'include' in th:
             append_children(th['include'], basedir, k, parent_type, results)
         elif 'include_tasks' in th:
-            append_children(th['include_tasks'], basedir, k, parent_type, results)
+            if 'file' in th['include_tasks']:
```

Comment:
To much duplication.
```suggestion
            append_children(
                th['include_tasks'].get('file', th['include_tasks']),
                basedir,
                k,
                parent_type,
                results
            )
```